### DISCUSSION TITLE
Start Dialog or Mainmenu

### DISCUSSION THREAD

**Discussion by ProgrammerLP** [LinwoodDev/Butterfly#286] [Polls]
With this poll I want to find out what kind of menu the Linwood community prefers. I ask you to vote honestly and not just vote for the usual variant, but take the time to think about it. Thank you for voting! I am looking forward to the result. 
NOTE: if you voted for mainmenu, please do the following: please write down what you think a perfect main menu would look like. Thanks for voting.

**Comment by Shahin-rmz** [comment]
You have picture examples?

**Reply by Shahin-rmz** [reply]
I don't exactly know what is dialogue what is simple main menu

**Reply by ProgrammerLP** [reply]
hello 👋,
at the moment i dont want to share ideas, because i want to hear ideas and feedback from the community how they imagine a main menu.
and based on that i make sketches and so and then present them ok?
programmerlp

**Reply by Shahin-rmz** [reply]
Good idea, Linwood knows, I personally have lack of front end experience. 
But my philosophy is that, less is better.
So, if we have a menu, we may click less to go to the app.
So menu is better than dialogue.

**Reply by ProgrammerLP** [reply]
i agree, i am totally on your side. do you have any ideas?

**Comment by CodeDoctorDE** [comment]
An idea for a main menu, what do you think?

![Screenshot_20220915-094655.jpg](https://user-images.githubusercontent.com/20452814/190348984-5b193192-7b5a-472f-8c8c-7fd3e267f6bc.jpg)
